alf/algorithms/predictive_representation_learner.py

```diff
@@ -97,6 +98,11 @@ def train_step(self, repr, state=()):
         return AlgStep(
             output=predicted_reward, state=state, info=predicted_reward)
 
+    def predict_step(self, repr, state=()):
+        predicted_reward = self._decoder_net(repr)[0]
```

Comment:
predicted_target. Also please update line 97 and line 99

Reply:
Changed